geode-core/src/test/java/org/apache/geode/internal/cache/tier/sockets/ClientHealthMonitorJUnitTest.java

```diff
@@ -35,7 +35,7 @@
 @Category(UnitTest.class)
 public class ClientHealthMonitorJUnitTest {
   public RestoreSystemProperties restoreSystemProperties = new RestoreSystemProperties();
-  private int pingIntervalMillis = 100;
+  private int pingIntervalMillis = 200;
```

Comment:
I think this will probably need to be much higher to never see failures in concourse - I think we could have JVM pauses of multiple seconds if GC hits at the wrong time.